packages/ng/establishment/src/lib/select/input/establishment-select-input.component.ts

```diff
@@ -54,7 +57,11 @@ export class LuEstablishmentSelectInputComponent<D extends ILuEstablishment = IL
 		protected _elementRef: ElementRef,
 		protected _viewContainerRef: ViewContainerRef,
 		protected _renderer: Renderer2,
-		@Inject(LuEstablishmentSelectInputIntl) public intl: ILuEstablishmentSelectInputLabel,
+		@Inject(ALuLegalUnitService) @Optional() @SkipSelf() hostLuService: ALuLegalUnitService,
+		@Inject(ALuLegalUnitService) @Self() selfLuService: LuLegalUnitService,
```

Comment:
@lucienbertin Juste par curiosité, pourquoi a-t-on besoin d'injecter certains services deux fois ?

Reply:
english

it is injected twice because we want the service to be overridable but also provide a solution that just works